src/common/components/MultiSelect.jsx

```diff
@@ -75,11 +87,41 @@ export default function MultipleSelectCheckmarks({
           multiple
           value={filterQuery[filterKey]}
           onChange={handleChange}
+          onClose={() => setSearchText("")}
```

Comment:
`() => setSearchText("")` is currently done at 2 places in the code. Probably a function called `resetSearchText` can be created and used at both places. Something like
```js
const resetSearchText = () => {
  setSearchText("");
}
```

```suggestion
          onClose={resetSearchText}
```